### ISSUE TITLE
Missing API declaration for NSPrivacyAccessedAPICategoryFileTimestamp

### ISSUE THREAD

**Issue by alekkova** [aws-amplify/amplify-ui-swift-liveness#126]
### Describe the bug

When submitting app binary for review we receive an automated warning from Apple:

>ITMS-91053: Missing API declaration - Your app’s code in the “App” file references one or more APIs that require reasons, including the following API categories: NSPrivacyAccessedAPICategoryFileTimestamp. While no action is required at this time, starting May 1, 2024, when you upload a new app or app update, you must include a NSPrivacyAccessedAPITypes array in your app’s privacy manifest to provide approved reasons for these APIs used by your app’s code.

AmplifyUILiveness depends on `https://github.com/aws-amplify/amplify-swift` version `2.27.2`, which does not declare the usage for `NSPrivacyAccessedAPICategoryFileTimestamp`. This has been fixed in `2.27.3` -> https://github.com/aws-amplify/amplify-swift/releases/tag/2.27.3 . Please update the dependency. 

### Steps To Reproduce

```swift
Steps to reproduce the behavior:
1. Submit app binary for Apple review.
2. Receive automated warning from Apple.
```

### Expected behavior

No warning from Apple.

### Swift Liveness Version

1.2.9

### Xcode version

15.2

### Is this a regression?

No

### OS Version

iOS 17

### Device

iPhone 15

**Comment by phantumcode** [comment]
@alekkova Thanks for submitting the issue.  We'll triage the issue and provide updates here.

**Comment by phantumcode** [comment]
Updates available in version [1.2.10](https://github.com/aws-amplify/amplify-ui-swift-liveness/releases/tag/1.2.10)
